elasticsearch/templates/statefulset.yaml

```diff
@@ -48,8 +48,10 @@ spec:
       {{- if .Values.schedulerName }}
       schedulerName: "{{ .Values.schedulerName }}"
       {{- end }}
+{{- if .Values.podSecurityContext }}
       securityContext:
-        fsGroup: {{ .Values.fsGroup }}
```

Comment:
Removing this is a backwards incompatible change. We can deprecate it, however anyone who already has it set should still be able to upgrade. 

My suggestion would be:

1. Add it back in underneath the `{{ toYaml .Values.podSecurityContext | indent 8 }}` that has been added below.  
2. Set fsGroup to an empty value in values.yaml
3. Leave it in the readme but mark it as a deprecated option. 

This will keep the change backwards compatible for anyone who has it set to a custom value.

Reply:
Changes applied! Not sure what happens if both values are set however.